modules/efk-logging-collector-fluentd-v-rsyslog.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-fluentd.adoc
+
+[id="efk-logging-collector-fluentd-v-rsyslog_{context}"]
+= About Fluentd and Rsyslog
+
+You can choose between Fluentd and Rsyslog as the logging collector. 
+
+Some of the noteable differences between the two collectors are:
+
+* Daching differences.
+
+* Deleted namespace
```

Comment:
I think these two items should be deleted.  The rsyslog Kubernetes metadata caching now, by default, works exactly like the fluentd Kubernetes metadata caching, and the parameters should be documented elsewhere (if any, if we have documented the fluentd Kubernetes metadata cache parameters).